src/main/java/com/entropy/entropay/auth/service/impl/JwtUserDetailsServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.entropy.entropay.auth.service.impl;
+
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+/*
+TODO: We will need to discuss how to load user details through this service.
```

Comment:
please sync up with @agustinb4 and Daniela. Do not create TODO without a ticket to track it